src_testbed/testbed.rs

```diff
@@ -419,6 +382,7 @@ impl Testbed {
     }
 
     fn clear(&mut self, window: &mut Window) {
+        //FIXME: do we need to do this still, after moving to harness code?
         self.callbacks.clear();
```

Comment:
Now that the harness is responsible for running the callbacks, all the callbacks-related code should be removed from the Testbed (and it turns out that the callbacks stored in the testbed are never called now anyways).

Reply:
I think this will require either:

* updating the examples code to run callbacks on the harness
* having a forwarding call on testbed to harness

I image the first is preferred, I will look into this.

Reply:
OK, looking now:

There is no way for callbacks to access the Window object through the harness. I'm not sure if this is an issue (probably a rare use case), but it may be..?

I will remove the callback code for now, perhaps it can be moved to a CallbackPlugin (that code still works I think), for use cases where it is needed.